cla_public/apps/checker/session.py

```diff
@@ -145,6 +145,20 @@ def aged_60_or_over(self):
     def callback_time(self):
         return self.get('CallMeBackForm_time')
 
+    @property
+    def eligibility_status(self):
+        status = 'unknown'
+        is_eligible = self.get('is_eligible', ELIGIBILITY_STATES.UNKNOWN)
+
+        if is_eligible == ELIGIBILITY_STATES.YES:
+            status = 'eligible'
+        elif is_eligible == ELIGIBILITY_STATES.NO:
+            status = 'ineligible'
+        elif is_eligible == ELIGIBILITY_STATES.UNKNOWN:
```

Comment:
Ah, the initial `status` should be empty string. The idea is that the status only becomes unknown when the at least one form is submitted. Unfortunately this includes the details form, which I'm hoping can be fixed separately in near future.